tests/queries/0_stateless/02483_add_engine_full_column_to_system_databases.sql

```diff
@@ -0,0 +1,5 @@
+DROP DATABASE IF EXISTS replicated_database_test; 
+CREATE DATABASE IF NOT EXISTS replicated_database_test ENGINE = Replicated('some/path/' || currentDatabase() || '/replicated_database_test', 'shard_1', 'replica_1');
```

Comment:
```suggestion
CREATE DATABASE IF NOT EXISTS replicated_database_test ENGINE = Replicated('some/path/' || currentDatabase() || '/replicated_database_test', 'shard_1', 'replica_1') SETTINGS max_broken_tables_ratio=1;
```
Let's also add some setting just to check that `SETTINGS` clause is printed as well.

Reply:
Updated